src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    mens = mens.filter(men => Math.ceil(men.died / 100) === century);
+  }
+
+  const mensYearsOfLife = mens.map(men => men.died - men.born);
+
+  const sum = mensYearsOfLife.reduce((accumulator, current) => {
+    return accumulator + current;
+  });
```

Comment:
use arrow function if it's possible

```suggestion
  const menAgesSum = mensYearsOfLife.reduce((sum, current) => sum + current);
```